examples/compiler/00_index.ipynb

```diff
@@ -0,0 +1,51 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The Devito compiler\n",
+    "\n",
+    "The following series of notebooks will describe these compilation passes in a hands-on fashion. We will explore the heart of the `Operator` class, which orchestrates code generation and compilation. For more details, see the [Operator class](https://github.com/opesci/devito/blob/master/devito/operator.py).\n",
+    "\n",
+    "In Devito, an `Operator` carries out three fundamental tasks: generation of low-level code, JIT compilation, and execution. Overall, the lowering process performed by an `Operator` – from high-level equations to dynamically compiled and executable code – consists of multiple compiler passes, summarized in the Figure bellow. \n",
+    "\n",
+    "![Compiler Stages](https://ia601500.us.archive.org/13/items/compiler-stages/compiler-stages.pdf)\n",
+    "\n",
+    "Many of the examples and exercises shown here are extracted from test cases found in the folder `devito/tests`, or from the documentation itself.\n",
+    "\n",
+    "## Outline\n",
+    "- 01) [Indexification](https://github.com/opesci/devito/blob/master/examples/compiler/01_indexification.ipynb)\n",
```

Comment:
the notebook names do not match. Maybe call this `01_lowering_indexification`, and change the filename from 01_lowering into 01_lowering_indexification ?

Reply:
typo! thank you, fixed